packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/auth-cognito-stack-builder.ts

```diff
@@ -678,7 +683,7 @@ export class AmplifyAuthCognitoStack extends cdk.Stack implements AmplifyAuthCog
   /**
    * Creates custom lambda to update userPool client on Cognito
    */
```

Comment:
Nit: This can be updated to reflect what this method is doing now

Reply:
We should also take away the two allow actions `'cognito-idp:CreateUserPoolDomain', 'cognito-idp:DescribeUserPool'` from the policy